marklogic-data-hub/src/test/java/com/marklogic/hub/scaffolding/CreateStepTest.java

```diff
@@ -52,8 +52,11 @@ void ingestionStep() throws IOException {
         verifyArtifactsAreWrittenToDbAndProject("myIngest2", "ingestion", null);
     }
 
+    // runAsDataHubDeveloper() runs as "flow-developer-role" in 9.0-x who doesn't have "hub-central-match-merge-writer, so explicitly
```

Comment:
It's not the role that's needed, it's the http://marklogic.com/data-hub/privileges/write-match-merge privilege, right? I think it's totally fine to add those privileges to flow-developer-role, I think that's the right fix here. flow-developer-role is already over-powered since it has the manage-admin role, there's no harm in giving it more DHF privileges.